src/psyclone/psyir/tools/dependency_tools.py

```diff
@@ -41,16 +41,91 @@
     dependency analysis.'''
 
 from __future__ import absolute_import, print_function
+from enum import IntEnum
+
+from sympy import Symbol, Integer
```

Comment:
Can you use "import sympy" and then use sympy.Symbol and sympy.Integer in the code.
Reading the code I was confused thinking you were creating PSyIR Symbols.

Reply:
Done.